src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```
якщо батьківський клас реалізує інтерфейс це означає, що і всі наслідники автоматично будуть його реалізувати